app/controllers/socure_webhook_controller.rb

```diff
@@ -52,18 +53,46 @@ def verify_queue(authorization_header:)
   end
 
   def log_webhook_receipt
-    event = socure_params[:event]
+    @event = socure_params[:event]
 
     analytics.idv_doc_auth_socure_webhook_received(
-      created_at: event[:created],
-      customer_user_id: event[:customerUserId],
-      event_type: event[:eventType],
-      reference_id: event[:referenceId],
-      user_id: event[:customerUserId],
+      created_at: @event[:created],
+      customer_user_id: @event[:customerUserId],
+      event_type: @event[:eventType],
+      reference_id: @event[:referenceId],
+      user_id: @event[:customerUserId],
+    )
+  end
+
+  def process_webhook_event
+    case @event[:eventType]
+    when 'DOCUMENTS_UPLOADED'
+      increment_rate_limiter
+    end
+  end
+
+  def increment_rate_limiter
+    @document_capture_session = DocumentCaptureSession.find_by(
+      user_id: @event[:customerUserId],
```

Comment:
The customerUserId will not be the user_id 🤪 - you can just search on the token.
```suggestion
```